egress_proxy/outputs.tf

```diff
@@ -3,12 +3,21 @@ output "https_proxy" {
   sensitive = true
 }
 
+output "http_proxy" {
+  value     = local.http_proxy
+  sensitive = true
+}
+
 output "domain" {
   value = local.domain
 }
 
-output "port" {
```

Comment:
Should we leave `port` with the same value as `https_port` for back-compat/migration purposes?